flink-end-to-end-tests/test-scripts/elasticsearch-common.sh

```diff
@@ -26,7 +26,7 @@ function setup_elasticsearch {
     mkdir -p $TEST_DATA_DIR
 
     local downloadUrl=$1
-    local elasticsearch_version=${2-0}
+    local elasticsearch_version=$2
```

Comment:
To make sure the value is actually set:

```suggestion
    local elasticsearch_version=$2

    if [ -z $elasticsearch_version ]; then
      echo "Elasticsearch version not declared."
      exit 1
    fi
```